packages/types/src/index.ts

```diff
@@ -150,6 +161,14 @@ export type TransferConfiguration = Readonly<{
    * The asset to be transferred.
    */
   destinationAsset: Asset;
+  /**
+   * [Optional] The position to launch the Meso experience. Defaults to `Position.TOP_RIGHT`.
+   */
+  position?: Position;
+  /**
+   * [Optional] The number of pixels (>= 0, integer) from the edges of the window to position the Meso experience. Defaults to `0`.
+   */
```

Comment:
nit: `[Optional]` is inferred by the `?`. We don't need to put it in the docs string.

---

For the `offset` docs string, I think we should clarify it a bit more. Maybe something like:

```
An positive integer as a string (excluding units) representing the number of pixels from the edges of the viewport to position the Meso experience. Defaults to `0`.
```